### PR TITLE
Add configurationClass option

### PR DESCRIPTION
**Pull request by aleswita**
all you can see in diff

### CODE REVIEW

File: src/DI/OrmExtension.php [hunk 1]
```diff
@@ -53,11 +55,18 @@ public function loadConfiguration(): void
 	public function loadDoctrineConfiguration(): void
 	{
 		$builder = $this->getContainerBuilder();
+
 		$config = $this->validateConfig($this->defaults['configuration'], $this->config['configuration']);
 		$config = Helpers::expand($config, $builder->parameters);
 
+		$configurationClass = $this->config['configurationClass'];
+
+		if ($configurationClass !== Configuration::class && !is_subclass_of($configurationClass, Configuration::class)) {
```
Comment (f3l1x) [inline src/DI/OrmExtension.php]: Have you considered `is_a` (http://php.net/manual/en/function.is-a.php)? 

```php
if (!is_a($configurationClass, Configuration::class)) {
// throw
}
```

File: tests/cases/DI/OrmExtensionTest.php [hunk 2]
```diff
@@ -51,13 +52,15 @@ public function testOwnEntityManager(): void
...
 					'entityManagerDecoratorClass' => DummyEntityManagerDecorator::class,
+					'configurationClass' => DummyConfiguration::class,
 				],
 			]);
 		}, self::class . __METHOD__);
 
 		/** @var Container $container */
 		$container = new $class();
 		self::assertInstanceOf(DummyEntityManagerDecorator::class, $container->getByType(DummyEntityManagerDecorator::class));
+		self::assertInstanceOf(DummyConfiguration::class, $container->getByType(DummyConfiguration::class));
```
Comment (f3l1x) [inline tests/cases/DI/OrmExtensionTest.php]: Please, add also test for the exception case. Thanks.

File: src/DI/OrmExtension.php (hunk 1 above)
Comment (aleswita) [reply src/DI/OrmExtension.php]: tnx for hint

File: tests/cases/DI/OrmExtensionTest.php (hunk 2 above)
Comment (aleswita) [reply tests/cases/DI/OrmExtensionTest.php]: Added.. can you help make test without warnings about deprecated anotations? I have no experience with PHPUnit.

Comment (f3l1x) [reply tests/cases/DI/OrmExtensionTest.php]: Replace annotation with method call.

https://github.com/nettrine/orm/commit/9d9413a97a9929bdc53f4592f98469962cfc354d#diff-9bd459dedc49d13726a368c5307d2fcaR41

Comment (aleswita) [reply tests/cases/DI/OrmExtensionTest.php]: done :-)

**Comment by f3l1x** [comment]
Awesome. 👍

**Pull request merged**